scripts/generateSlimMetadata.ts

```diff
@@ -0,0 +1,80 @@
+// Running this file with ~ yarn generate-slim-metadata will create slim metadata
+import {Api} from '../packages/api/src/Api';
+import MetadataVersioned from '@polkadot/metadata/Metadata/MetadataVersioned';
+
+import {createType} from '@cennznet/types';
+
```

Comment:
would be nice to add some description of what it's removing etc.

Reply:
Or possibly what we are keeping, as it seems we are better aware of the parts that we are interested in.

Reply:
thanks people.. will update

Reply:
made an attempt to describe.. let me know if it requires edition..